### DISCUSSION TITLE
[Feature Request]: Bot订阅时增添规则

### DISCUSSION THREAD

**Discussion by XiaYi1002** [jxxghp/MoviePilot#3416] [讨论]
### 当前程序版本

1.3.5

### 功能改进类型

Docker

### 功能改进

目前订阅流程，它没有问题，就是说，可否在回复 序号 进行订阅这一步，添加一些想要设定的规则，比如，包含、排除、站点。
既然选择了通过bot去订阅，那么此时我的想法那就是不去打开这个WEB页面，所以两者之前就存在一定的矛盾。然后就是 Bot查询订阅，它是需要输入 一串英文指令 [序号]，就是说可否改成中文的，英文确实有点小长，对于英文不咋地的我，打字又慢的我，确实是一个期望改进的点；再者就是历史记录，可否增添一个 状态 筛选，因为有时候一时间增添的剧集比较多，然后要找失败的，都还要翻好久，日常的话，还好，但同样的问题，处在于，刚初始化的时候，那会好多转移失败的，我要一页一页去翻，着实是翻了好久。开ALL的话，页面又卡的动不了。

省流助手：
1、订阅增添 包含、排除、站点定义
2、英文指令转为中文
3、历史记录增加状态筛选

//站点这一块，不晓得他人是怎么想的，但我的小思路就是，如果在选择站点的时候，能直接绑定该站点管组，是否更好一些呢，比如绑定 呆逼 那么呆逼的管组名称就是 DDWEB ，那么我们在订阅的时候，选择了 呆逼，订阅的规则里就会自动添加 DDWEB这个条件

### 参考资料

一个资源一个站点一个品质，这是我的习惯，所以我自己折腾的，根据自己的需求，弄了这个一个订阅的bot。
1-发送 搜索 [资源名称] 查找资源
2-发送 订阅 序号,季数,包含,排除,站点（多个 包含、排除、站点 值，用&隔开) 进行资源订阅

![Snipaste_2023-10-19_20-11-59](https://github.com/jxxghp/MoviePilot/assets/24986956/2b3e8570-40fe-4d58-82a4-59ea5baceb61)

**Comment by xwndo** [comment]
请问这个订阅bot如何弄的呀

**Comment by XiaYi1002** [comment]
> 请问这个订阅bot如何弄的呀

自己写的

**Comment by weidongzhou1994** [comment]
这个能分享一下吗?
